react/features/base/tracks/middleware.web.ts

```diff
@@ -121,11 +127,40 @@ MiddlewareRegistry.register(store => next => action => {
         return result;
     }
 
+    case CONFERENCE_JOINED: {
+        _addSetCameraFacingModeListener(action.conference);
+        break;
+    }
     }
 
     return next(action);
 });
 
+/**
+ * Registers listener for {@link JitsiConferenceEvents.ENDPOINT_MESSAGE_RECEIVED} that
+ * will perform various chat related activities.
+ *
+ * @param {IJitsiConference} conference - The conference.
+ * @returns {void}
+ */
+function _addSetCameraFacingModeListener(conference: IJitsiConference) {
```

Comment:
Can you please move this to features/conference ? Generally speaking "base" features add blocks for other features to build on top, generally without UI, and actual features use them.